README.md

```diff
@@ -1 +1,43 @@
-# node-modules-json-lab
\ No newline at end of file
+# Importing JSON Files Lab
+
+## Objectives
+
+1. Implement a JSON config file 
+1. Implement file which imports JSON
+1. Use JSON in the file/script
+
+## Introduction
+
+In this lab, you'll learn how to import JSON files. This can be useful for data (e.g., list of customers) or configuration (e.g., server settings). In other words, JSON files are good for storing static information, i.e., information without JavaScript/Node code.
+
```

Comment:
What specifically are the going to be working on now? This is a good intro for reminding them why they need to learn to work with JSON, but let's also be specific about the type of work theyre doing in this lab